image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -92,7 +92,7 @@
         {{- $_ := set $._rox.scannerV4.indexer.serviceTLS "generate" true -}}
       {{- end -}}
     {{- end }}
-    {{ $cryptoSpec := dict "CN" "SCANNER_SERVICE: Scanner V4 Indexer" "dnsBase" "scanner-v4" }}
+    {{ $cryptoSpec := dict "CN" "SCANNER_SERVICE: Scanner V4 Indexer" "dnsBase" "scanner-v4-indexer" }}
```

Comment:
I thought we also have `SCANNER_V4_INDEXER_SERVICE`. Is that not needed, too?

Reply:
I agree, it is my understanding that the initial uppercase identifier needs to correspond to the entries in https://github.com/stackrox/stackrox/blob/5cd9f69072c17146661fe63bfda293fa08b10003/proto/storage/service_identity.proto#L21. Same for matcher.

Reply:
Excellent, was curious about that, updated the service types for indexer/matcher/and DB